### PR TITLE
Ingesting data for payment type : ACH Debit

### PR DESCRIPTION
**Pull request by pranjaldayama**

### CODE REVIEW

**Comment by pranjaldayama** [comment]
Tested the change by ingesting data successfully to environment rough-dream-11
http://rough-dream-11-editorial.backbase.test:8080/gateway/business-banking-demo/login

image

image

File: src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java [hunk 1]
```diff
@@ -57,8 +60,10 @@ public void ingestPaymentOrders(String externalUserId) {
 
             if (PAYMENT_TYPE_SEPA_CREDIT_TRANSFER.equals(paymentType)) {
                 randomArrangement = getRandomFromList(sepaCtArrangements);
-            } else {
+            } else if (PAYMENT_TYPE_US_DOMESTIC_WIRE.equals(paymentType)) {
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java]: add `|| PAYMENT_TYPE_US_FOREIGN_WIRE` to the condition

File: src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java [hunk 2]
```diff
@@ -57,8 +60,10 @@ public void ingestPaymentOrders(String externalUserId) {
...
                 randomArrangement = getRandomFromList(usDomesticWireArrangements);
+            } else {
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java]: it's better to make this an `else if (PAYMENT_TYPE_ACH_DEBIT.equals(paymentType)) {`
and `throw new IllegalArgumentException("Unknown payment type " + paymentType);` in the `else` branch

File: src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java (hunk 1 above)
Comment (pranjaldayama) [reply src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java]: done :)

File: src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java (hunk 2 above)
Comment (pranjaldayama) [reply src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java]: done :)

**Pull request merged**